.drone.yml

```diff
@@ -16,3 +16,14 @@ steps:
     image: node:8.10-alpine
     commands:
       - yarn test-ci
+
+  - name: npm
```

Comment:
Are there no commands for this job?

Reply:
http://plugins.drone.io/drone-plugins/drone-npm/